### DISCUSSION TITLE
Rename of repo

### DISCUSSION THREAD

**Discussion by jeberhardt** [cds-snc/url-shortener#38] [Polls]
I'd like to suggest renaming this repo, to better reflect its position within CDS (ie: platform), and also the functionality scope of this repo (ie: API). Forms uses the format `platform-forms-client`, and following that, I'd like to suggest `platform-url-shortener-api`.

Related repos would include `platform-url-shortener-client` and `platform-url-shortener-terraform`.

**Comment by sylviamclaughlin** [comment]
My 2cents but please feel free to ignore. I think first we have to make the decision whether we will be separating these 2 products into a separate api and a separate frontend client. Right now, the frontend is within the the url-shortener product. Once we fully decide if it will be 2 separate repos and essentially products, then I think it will make more sense in renaming. I am open to both suggestions,  but of course we all have to fully decide. Also, I have usually seen the terraform code live within the project and not a separate repo but I am not sure what is the CDS standard - maybe Pat can chime in?

**Reply by dinophile** [reply]
I think two separate repos makes sense. It will be a bit easier to reason about rather than combine them. It would be my vote. For what that's worth now as the PM! 😂 

At least from personal experience, detangling a bug is so much easier if you keep your contexts separate.

**Reply by sylviamclaughlin** [reply]
Sounds good to me! I was leaning towards that too since it will be easier overall for maintainability.
